shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java

```diff
@@ -236,6 +241,26 @@ public <S> S getFromGateway(final String path, final Map<String, Object> headers
         }
     }
 
+    /**
+     * Send a get http request to shenyu gateway .
+     *
+     * @param <S>     response type
+     * @param path    path
+     * @param type    type of response passed to {@link Gson#fromJson(String, Type)}
+     * @return response from gateway
+     * @throws IOException IO exception
+     */
+    public <S> S testMetricsPluginFromGateway(final String path, final Type type) throws IOException {
```

Comment:
We can rename `testMetricsPluginFromGateway` to `getHttpService`, and a param `headers` can be added for more general use.

Reply:
good idea，thanks～